apps/web/components/Layout.tsx

```diff
@@ -7,19 +7,24 @@ type LayoutProps = {
   size?: Size;
   hover?: boolean;
   className?: string;
+  title?: string;
 };
 
 export const Layout = ({
   children,
   size = "md",
   hover = false,
   className,
+  title,
 }: LayoutProps) => {
   return (
-    <div
-      className={`border1 cursor-pointer bg-white transition-all duration-200 ease-in-out ${hover && "hover:bg-surfaceHover hover:border-borderHover"} ${classNames({ "p-4": size === "sm", "rounded-[16px] p-6": size === "md", "rounded-[24px] p-8": size === "lg" })} ${className}`}
+    <section
+      className={`border1 relative bg-neutral transition-all duration-200 ease-in-out ${hover && "hover:bg-secondary-soft cursor-pointer hover:border-secondary-content"} ${classNames({ "p-4": size === "sm", "rounded-[16px] p-6": size === "md", "rounded-[24px] p-8": size === "lg" })} ${className}`}
```

Comment:
maybe we should extract some to variables, this line is too much complicated to understand if we ever have a class problem to fix

Reply:
We dont use this component anymore. We use class section-layout and pout all the classnames there